classes/Task/Update/UpdateFiles.php

```diff
@@ -196,6 +196,12 @@ protected function warmUp(): int
             $this->container->getCompletionCalculator()->getBasePercentageOfTask(self::class)
         );
 
+        $archiveXml = $this->container->getUpdateConfiguration()->getLocalChannelXml();
+        $this->container->getFileLoader()->addXmlMd5File(
+            $this->container->getUpgrader()->getDestinationVersion(),
+            $this->container->getProperty(UpgradeContainer::DOWNLOAD_PATH) . DIRECTORY_SEPARATOR . $archiveXml
+        );
+
```

Comment:
this is not only necessary for the local channel

```suggestion


         if ($this->container->getUpdateConfiguration()->isChannelLocal()) {
            $archiveXml = $this->container->getUpdateConfiguration()->getLocalChannelXml();
            $this->container->getFileLoader()->addXmlMd5File($this->container->getUpgrader()->getDestinationVersion(), $this->container->getProperty(UpgradeContainer::DOWNLOAD_PATH) . DIRECTORY_SEPARATOR . $archiveXml);
        }

```

Reply:
Forgot to commit it, indeed